src/main/java/org/spongepowered/api/service/command/SimpleCommandService.java

```diff
@@ -231,8 +211,8 @@ public boolean call(CommandSource source, String arguments, List<String> parents
     }
 
     @Override
-    public boolean testPermission(CommandSource source) {
-        return dispatcher.testPermission(source);
+    public boolean testPermission(CommandSource source, boolean ignorePermissions) {
```

Comment:
Would an entirely new method not be more appropriate?

testPermissions(ignorePermissions = true) seems a bit roundabout.